platform/felib/ui.apps/src/main/content/jcr_root/apps/felib/components/lib/js.html

```diff
@@ -1,3 +1,3 @@
-<sly data-sly-use.model="com.peregrine.felib.models.JSModel"/><sly data-sly-list="${model.files}">
+<sly data-sly-use.model="com.peregrine.felib.models.JSModel" data-sly-list="${model.files}">
 <!-- ${item.path} -->
 ${item.content @ context='unsafe'}</sly>
```

Comment:
`</sly>`
in new line to make clean-clean?

Reply:
should look better now